### PR TITLE
Return name of support menu back to Wiki, and Archive to Guides

### PR DESCRIPTION
**Pull request by ctzsnooze**
This PR builds on the excellent re-organisation of the documents in PR #269 and related work.

Changes made:
- reverts the Support menu title back to Wiki,
- re-names the Archive folder to Guides, sorts them into Current and Archive (old)
- includes the minor internal link syntax error fix in PR270,
- welcome to wiki changes
- Minor CSS changes (H1 headings not so big, font 100% just to suit my screen can be fiddled with)
- multiple small fixes to documents with images, old documents with warnings to modern format.
- fixed internal links in documents with old-style Github internal links, which permitted capitals, whereas Docusaurus does not
- all .mdx files re-named to .md since this was the only way to get past the `check_file_names.mjs` checks, which prevented commits.  Eventually I found out that any file in docs with .mdx extension would fail.  Even a file with nothing but a valid heading.  So I re-named all failing files to .md extension and then all checks passed.

All files load into Docosaurus with no errors.  

See below for images.

### CODE REVIEW

**Comment by nerdCopter** [comment]
?? Regarding another comment, I thought you recommended "`Support/Wiki`" as the menu item, not revert to only "`Wiki`". I prefer the former if possible.

**Comment by ctzsnooze** [comment]
@nerdCopter That was my original suggestion.  The problem with Support > Wiki is that not a lot of stuff ends up in 'Wiki'.  

`Wiki` kind of encompasses all the 'user-focused' documentation that we have.  By naming the Menu, `Wiki`, we are then free to subdivide it into meaningful chunks.  

And we may offer true 'support' services at some point, so it may be best to retain that name for that specific purpose.

**Comment by ctzsnooze** [comment]
This is how it looks now with this PR.  
Screen Shot 2023-12-16 at 22 41 38

The files in 'Guides' are being sorted into 'Current' and 'old', or 'Archive' groups.
Screen Shot 2023-12-16 at 22 39 02

**Comment by SupaflyFPV** [comment]
Great effort, I'd been meaning to go through a lot of this and update as well...can't believe its the end of the year already...will review asap

File: docusaurus.config.js [hunk 1]
```diff
@@ -132,34 +132,28 @@ const config = {
...
+          //     sidebarId: 'release',
+          //     position: 'left',
+          //     label: 'Release',
+          //   },
+          //   {
+          //     type: 'docSidebar',
+          //     sidebarId: 'tuning',
+          //     position: 'left',
+          //     label: 'Tuning',
+          //   },
```
Comment (haslinghuis) [inline docusaurus.config.js:156]: Can probably be removed

File: docs/development/building/Building-in-Eclipse.md [hunk 2]
```diff
@@ -59,72 +59,72 @@ If you'll be submitting changes to betaflight, [fork the repository](https://hel
 
 In Eclipse go to _File -> Import_ choose _Git -> Projects from Git_
 
-![projects from git](assets/building-in-eclipse/checkout-betaflight-001.PNG)
+![projects from git](../assets/building-in-eclipse/checkout-betaflight-001.PNG)
```
Comment (haslinghuis) [inline docs/development/building/Building-in-Eclipse.md:62]: Is there a way to have a direct path (symlink)?

File: docs/relations/partner.md [hunk 3]
```diff
@@ -1,4 +1,11 @@
-import PartnerText from './../../docs/manufacturer/approved-partner-program.mdx'
+---
+sidebar_position: 1
+sidebar_label: Approved Partner Program
+---
+
+import PartnerText from './../../docs/development/manufacturer/approved-partner-program.md'
```
Comment (haslinghuis) [inline docs/relations/partner.md:6]: This was here before - can we have a direct path instead for better maintenance?

**Review by haslinghuis — DISMISSED** [review]

**Review by VitroidFPV — APPROVED** [review]

**Comment by VitroidFPV** [comment]
Oh, that did reset the whole approval status again :/

**Review by haslinghuis — APPROVED** [review]

**Review by VitroidFPV — APPROVED** [review]

**Pull request merged**